Automattic/woocommerce-payments
# Migrate remaining UI tests to RTL

Fixes #763

#### Changes proposed in this Pull Request

* Migrate remaining UI test to React testing library.
* Remove enzyme dependency.

#### Testing instructions

See Travis checks. To test locally:

1. Run `npm install`.
2. Run `npm test`. All test should be green.

-------------------
